core/Plugin/LogTablesProvider.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Plugin;
+
+use Piwik\Container\StaticContainer;
+use Piwik\Tracker\LogTable;
+
+class LogTablesProvider {
```

Comment:
I named it LogTablesProvider as this is currently the naming in Piwik 3. As this won't be a public API I directly went for the Piwik 3 naming. We might change the naming later in Piwik 3 and move all providers into a subdirectory eg `Provider\LogTables` but then we can still refactor it